src/components/main-layout/Layout.tsx

```diff
@@ -33,21 +33,19 @@ const MainLayout: React.FC<MainLayoutProps> = ({
                 className={classNames([
                     styles.contentPanel,
                     {
-                        [styles.finalPage]: page === 11,
+                        [styles.finalPage]: page === 10,
```

Comment:
suggestion: create a Pages.ts module, hardcode symbolic names for each page and keep the numbers only in that module.  It could be an enum or a class or just a bunch of constants or whatever. Pages.ts can have things like nextPage(PAGE_FOO) and stuff like that.  Import Pages.ts everywhere....